src/variables/components/VariablesTab.test.tsx

```diff
@@ -0,0 +1,427 @@
+      message: 'Variable created successfully',
+      data: {
+        name: 'test_variable_name',
+        id: 'test_variable_id',
+        orgID: 'test_org_id',
+        arguments: {
+          type: 'map',
+          values: {
+            sample: 'value',
+          },
+        },
+      },
+    }
+  }),
+  patchVariable: jest.fn(() => {
+    return {
+      status: 200,
+      message: 'Variable updated successfully',
+      data: {
+        name: 'test_variable_name',
+        id: 'test_variable_id',
+        orgID: 'test_org_id',
+        arguments: {
+          type: 'map',
+          values: {
+            sample: 'value',
+          },
+        },
+      },
+    }
+  }),
+  deleteVariable: jest.fn(() => {
+    return {
+      status: 204,
+      message: 'Variable deleted successfully',
+      data: {
+        name: 'test_variable_name',
+        id: 'test_variable_id',
+        orgID: 'test_org_id',
+        arguments: {
+          type: 'map',
+          values: {
+            sample: 'value',
+          },
+        },
+      },
+    }
+  }),
+}))
+
+jest.mock('src/client/index.ts')
+jest.mock('src/shared/actions/notifications')
+jest.mock('src/resources/components/GetResources')
+jest.mock(
+  'src/annotations/components/overlay/CreateAnnotationStreamOverlay',
+  () => () => null
+)
+jest.mock('src/checks/components/NewThresholdCheckEO.tsx', () => () => null)
+jest.mock('src/checks/components/NewDeadmanCheckEO.tsx', () => () => null)
+jest.mock('src/resources/selectors/index.ts', () => {
+  return {
+    getAll: jest.fn(() => {
+      return []
+    }),
+    getLabels: jest.fn(() => {
+      return []
+    }),
+  }
+})
+
+// Mock State
+import {renderWithReduxAndRouter} from 'src/mockState'
+import {withRouterProps} from 'mocks/dummyData'
+import {mockAppState} from 'src/mockAppState'
+
+// Redux
+import VariablesIndex from '../containers/VariablesIndex'
+import {normalize} from 'normalizr'
+import {Organization} from '../../client'
+import {OrgEntities, RemoteDataState} from '../../types'
+import {arrayOfOrgs} from '../../schemas'
+import {fireEvent, cleanup, screen, waitFor} from '@testing-library/react'
+import {
+  OverlayController,
+  OverlayProviderComp,
+} from '../../overlays/components/OverlayController'
+
+import {mocked} from 'ts-jest/utils'
+import {notify} from 'src/shared/actions/notifications'
+import {
+  createVariableSuccess,
+  deleteVariableSuccess,
+  updateVariableSuccess,
+} from '../../shared/copy/notifications'
+
+const defaultProps: any = {
+  ...withRouterProps,
+  org: '',
+}
+
+const setup = (props = defaultProps) => {
+  return renderWithReduxAndRouter(
+    <>
+      <OverlayProviderComp>
+        <OverlayController />
+        <VariablesIndex {...props} />
+      </OverlayProviderComp>
+    </>,
+    _fakeLocalStorage => {
+      const appState = {...mockAppState}
+      return appState
+    }
+  )
+}
+
+let getByTestId
+let store
+let getByTitle
+
+describe('the variable integration tests', () => {
```

Comment:
we're not describing the tests, we're describing the thing under test - the variables ui functionality